stable/weave-cloud/update.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh -eux
```

Comment:
Glad someone is finally reviewing this!

What we have is this generator API that is normally used with `kubectl create -n kube-system `https://cloud.weave.works/k8s.yaml?t=<service_token>`, but for this chart we simply check-in the YAML generated by the API, hence this script here.